packages/webchat/src/typings.ts

```diff
@@ -19,9 +19,6 @@ declare global {
     USE_SESSION_STORAGE: boolean
     // TODO: this should be of type BPStorage
     BP_STORAGE: any
-    botpress: {
-      [moduleName: string]: any
-    }
```

Comment:
Reason behind this change?

Reply:
There's a declaration of botpress already in the components package